forest/shared/src/cli/config.rs

```diff
@@ -17,6 +17,17 @@ pub struct LogConfig {
     pub filters: Vec<LogValue>,
 }
 
+impl LogConfig {
+    pub(crate) fn to_filter_string(&self) -> String {
+        let filters: Vec<_> = self
+            .filters
+            .iter()
+            .map(|f| format!("{}={}", f.module, f.level))
+            .collect();
+        filters.join(",")
```

Comment:
one-combo opportunity, we can't miss it
```suggestion
        self
            .filters
            .iter()
            .map(|f| format!("{}={}", f.module, f.level))
            .collect::<Vec<_>>()
            .join(",")
```